common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -29,30 +29,42 @@ killall_program() {
     done
 }
 
+prplmesh_platform_db_init() {
+    management_mode=${1-Multi-AP-Controller-and-Agent}
+    operating_mode=${2-Gateway}
+
+    mkdir -p /tmp/@PLATFORM_DB_PATH_TEMP@/
```

Comment:
I must have missed something somewhere... `PLATFORM_DB_PATH_TEMP` seems to be only defined as a cpp macro, not as a CMake variable. Does this work? Doesn't this get substituted with just empty?

Also, the defintiion of the cpp macro already includes the /tmp part.

Reply:
We call it a "foox" in hebrew - meaning it is pure luck :)
@PLATFORM_DB_PATH_TEMP@ is indeed substituted to empty, so the directory is just /tmp which is why it works. I'll fix it.